tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCoverageLevel.java

```diff
@@ -1272,15 +1292,15 @@ public void testLSR() {
             }
             final CoverageStatus value = entry.getValue();
             if (value.inId) {
-                String[] parts = key.split("\t");
+                String[] parts = key.split("\t"); // split into language and code
                 ids.add(parts[1]);
                 if (!value.inRoot) {
                     missing.add(parts[1]);
                 }
             }
         }
         if (!assertEquals(
-                "Language subtags that are in a CLDR locale's ID are in root ("
+                "Language subtags in a locale's ID must be in one of the attributeValueValidity.xml $language* sets, typically $languageNonTcLtBasic  ("
```

Comment:
Is there also a test that warns if a locale is being removed from a TC org's language set that it needs to be changed to be in the non-TC org Basic or higher list?

Reply:
There isn't, and that's reasonable. But I'd like that to be a separate ticket, since we don't have to do it this release.

Reply:
Okay, SGTM!